komga-webui/src/views/Home.vue

```diff
@@ -97,6 +97,20 @@
         </v-list-item>
       </v-list>
 
+    <v-divider />
+
+    <v-list>
+        <v-list-item @click="toggleDarkMode">
+          <v-list-item-icon>
+            <v-icon>mdi-lightbulb</v-icon>
```

Comment:
The icon could change depending on the theme in use.

Maybe `mdi-brightness-7` when the light theme is on, and `mdi-brightness-3` when the dark theme is on.